rewriter/Prop.cc

```diff
@@ -379,7 +379,7 @@ vector<ast::ExpressionPtr> processProp(core::MutableContext ctx, PropInfo &ret,
         // via `T.assert_type!(self.class.compute_foo(T.unsafe(nil)), type)` in the getter.
         auto selfSendClass = ast::MK::Send0(computedByMethodNameLoc, ast::MK::Self(loc), core::Names::class_(),
                                             computedByMethodNameLocZero);
-        auto raiseUnimplemented = ast::MK::RaiseTypedUnimplemented(computedByMethodNameLoc);
+        auto raiseUnimplemented = ast::MK::RaiseUnimplemented(computedByMethodNameLoc);
```

Comment:
We're still not going to be able to use `computed_by` in `# typed: strong` files but there are only 1,000 or so of those, and I'm kind of find with that because this is janky anyways.